requirements-freeze.txt

```diff
@@ -50,13 +50,13 @@ pyutilib.component.core==4.5.3
 PyYAML==5.3
 redis==2.10.1
 repoze.lru==0.6
-repoze.who==1.0.18
+repoze.who==2.0
```

Comment:
Pretty sure we can't ship 2.0 to production because it breaks [ckanext-saml2](https://github.com/GSA/catalog-app/issues/78).